AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Utils/IdentifyTextResultTransformers.swift

```diff
@@ -137,15 +110,47 @@ class IdentifyTextResultTransformers: IdentifyResultTransformers {
             keyValues: keyValues)
     }
 
-    static func processTable(_ block: AWSTextractBlock,
+    static func processTables(tableBlocks: [AWSTextractBlock],
+                              blockMap: [String: AWSTextractBlock]) -> [Table] {
+        var tables = [Table]()
+        for tableBlock in tableBlocks {
+            if let table = processTable(tableBlock, blockMap: blockMap) {
+                tables.append(table)
+            }
+        }
+        return tables
+    }
+
+    static func processKeyValues(keyValueBlocks: [AWSTextractBlock],
+                                 blockMap: [String: AWSTextractBlock]) -> [BoundedKeyValue] {
+        var keyValues =  [BoundedKeyValue]()
+        for keyValueBlock in keyValueBlocks where keyValueBlock.entityTypes?.contains("KEY") ?? false {
```

Comment:
I think it makes sense to do something like:
move:
```
where keyValueBlock.entityTypes?.contains("KEY") ?? false
```
into  the function`processKeyValue()`:
```
 static func processKeyValue(_ keyBlock: AWSTextractBlock, .......... {
        guard keyBlock.entityTypes?.contains("KEY") ?? false,
            let relationships = keyBlock.relationships else {
            return nil
        }
   ...
}
```
((Note that the `keyBlock` parameter does not need to be Optional))